src/colorprint.cc

```diff
@@ -89,7 +89,11 @@ std::string FormatString(const char* msg, va_list args) {
 
   std::size_t size = 256;
   char local_buff[256];
+#ifdef __ANDROID__
+  auto ret = ::vsnprintf(local_buff, size, msg, args_cp);
```

Comment:
Tested this approach, doesn't work. This is quirk of Android GNU STL library. This problem doesn't appear if compiling with  `-DCMAKE_ANDROID_STL_TYPE=c++_static` which uses clang stl library. But gnustl is default for Android now. I can replace it with `#if defined(__ANDROID__) && defined(__GLIBCXX__)` - it works, if you think it is better.